build_projects/dotnet-cli-build/build.sh

```diff
@@ -101,6 +101,15 @@ then
     ulimit -n 1024
 fi
 
+# Temporary logic to turn core dumps on so we can catch exit code 138 errors on OSX
+if [ "$(uname -s)" == "Darwin" ]; then
+  # If there are no core files already in /cores/, we'll enable core dump
+  # generation for this shell.
+  if [ "$(ls -A /cores)" ]; then 
```

Comment:
> Is this necessary?

We do not want crash dumps to be generated once the CLI build exits.

> What do you want to see happening?

Capture contents of cores folder (before and after) in the log to determine if we caught anything or not.